pavelib/servers.py

```diff
@@ -135,6 +135,7 @@ def devstack(args):
     if args.optimized:
         settings = OPTIMIZED_SETTINGS
         asset_settings = OPTIMIZED_ASSETS_SETTINGS
+    sh(django_cmd('cms', settings, 'reindex_course', '--setup'))
```

Comment:
All the logic regarding ElasticSearch indexing resides in the CMS, since course staff members will be indexing courses from there. You can see that in cms/envs/devstack.py we have a flag `ENABLE_COURSEWARE_INDEX`.

In lms/envs/devstack.py are the settings for enabling searching. That's the reason why the command lives in the CMS.